crates/anvil/src/eth/api.rs

```diff
@@ -1665,6 +1666,14 @@ impl EthApi {
         Ok(self.miner.is_auto_mine())
     }
 
+    /// Returns the value of mining interval, if set.
+    ///
+    /// Handler for ETH RPC call: `anvil_getIntervalMing`
+    pub fn anvil_get_interval_ming(&self) -> Result<Option<u64>> {
```

Comment:
should this be `anvil_get_interval_mining` not `anvil_get_interval_ming`?

Reply:
thank you! yes indeed, missed the typo :grin: preparing an update in https://github.com/foundry-rs/foundry/pull/9292